File: API/ContainerNinja.Contracts/DTO/KitchenProductDTO.cs

using ContainerNinja.Contracts.Enum;

namespace ContainerNinja.Contracts.DTO
{
    public class KitchenProductDTO
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public float? Amount { get; set; }

        public KitchenUnitType KitchenUnitType { get; set; }

        public int? ProductId { get; set; }

        public WalmartProductDTO Product { get; set; }
    }
}

File: Client/src/app/models/KitchenProductDTO.ts
import { WalmartProductDTO } from "./WalmartProductDTO";

export class KitchenProductDTO {
  id?: number = undefined;
  name?: string = undefined;
  amount?: number = undefined;
  kitchenUnitType?: number = undefined;
  productId?: number | undefined = undefined;
  walmartProduct?: WalmartProductDTO = undefined;
}
